product_docs/docs/pgd/5/reference/conflicts.mdx

```diff
@@ -17,39 +17,41 @@ PGD recognizes the following conflict types, which can be used as the `conflict_
 -   `update_pkey_exists` &mdash; An incoming update has modified the `PRIMARY KEY` to a value that already exists on the node that's applying the change.
 -   `multiple_unique_conflicts` &mdash; The incoming row conflicts with multiple UNIQUE constraints/indexes in the target table.
 -   `delete_recently_updated` &mdash; An incoming delete with an older commit timestamp than the most recent update of the row on the current node or when using [row version conflict detection](../consistency/conflicts/03_conflict_detection/#row-version-conflict-detection).
+<!-- Missing verb>
 -   `delete_missing` &mdash; An incoming delete is trying to remove a row that doesn't exist.
 -   `target_column_missing` &mdash; The target table is missing one or more columns present in the incoming row.
 -   `source_column_missing` &mdash; The incoming row is missing one or more columns that are present in the target table.
 -   `target_table_missing` &mdash; The target table is missing.
 -   `apply_error_ddl` &mdash; An error was thrown by Postgres when applying a replicated DDL command.
 
-# Conflict resolution
+## Conflict resolution
 
-Most conflicts can be resolved automatically. PGD defaults to a last-update-wins mechanism or, more accurately, the `update_if_newer` conflict resolver. This mechanism retains the most recently inserted or changed row of the two conflicting ones based on the same commit timestamps used for conflict detection. The behavior in certain corner-case scenarios depends on the settings used for `bdr.create_node_group` and alternatively for `bdr.alter_node_group`.
+Most conflicts can be resolved automatically. PGD defaults to a last-update-wins mechanism or, more accurately, the `update_if_newer` conflict resolver. This mechanism retains the most recently inserted or changed row of the two conflicting ones based on the same commit timestamps used for conflict detection. The behavior in certain corner-case scenarios depends on the settings used for `bdr.create_node_group` and, alternatively, for `bdr.alter_node_group`.
 
 PGD lets you override the default behavior of conflict resolution by using the following function.
```

Comment:
Instead of saying "following function," should this say "conflict resolvers"? There's no function that follows.

Reply:
```suggestion
PGD lets you override the default behavior of conflict resolution by using the `bdr.alter_node_set_conflict_resolver()` function.
```